### PR TITLE
log out of converse when logging out of roundcube

### PR DESCRIPTION
**Pull request by HanakoSeishin**

### CODE REVIEW

File: js/converse-rcmail.js [hunk 1]
```diff
@@ -23,6 +23,14 @@ function rcmail_converse_init(converse, args)
             rcmail.local_storage_set_item('converse.rid', converse.tokens.get('rid'));
         }
     });
+    
+    // log out of converse when logging out of roundcube
+    rcmail.addEventListener('beforeswitch-task', function(e){
+        if(e == 'logout'){
+                //console.log('logout');
```
Comment (devurandom) [inline js/converse-rcmail.js]: Could you please remove this comment?

**Review by devurandom — CHANGES_REQUESTED** [review]

**Comment by devurandom** [comment]
Thanks! Can you please squash the two commits into one?

**Comment by HanakoSeishin** [comment]
Sorry, now idea how to do so. I'm totally new here and the whole interface is still confusing and overwhelming.

**Comment by devurandom** [comment]
The first result of the web search I did explains quite well how to do it: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

Ignore the warning about not doing this to pushed commits -- in this case this is actually what you want to do. When pushing the squashed commit, you will have to force-push it using `git push --force`, though.

**Comment by HanakoSeishin** [comment]
I'm terribly sorry, but explaining quite well how make a barrel roll doesn't help if one doesn't know how to take off in the first place. I was under impression that the purpose of this site, well at least part of it, is making it easy for people to contribute. But I guess I was wrong. Maybe I'll come back some day when I can get what that article is even talking about.

**Comment by devurandom** [comment]
You're right, the site is quite more advanced by now, than I realised. I can squash the commits for you from the web-interface.

**Comment by devurandom** [comment]
For whatever reason this did not go well, so I manually edited the history. Should be fine now.

Thanks for the contribution!

**Comment by mckaygerhard** [comment]
solution for to #10  and #5 thanks

**Pull request merged**